build/optaplanner-build-parent/pom.xml

```diff
@@ -526,8 +485,7 @@
             </execution>
             <execution>
               <id>ban-duplicated-classes</id>
-              <!-- Using phase=none as we don't want this execution as part of the default build. The phase
-                   is specified in the "full" profile and thus the execution will only be activated when using that profile. -->
+              <!-- Currently disabled as it fails the build. See https://issues.redhat.com/browse/PLANNER-2261. -->
```

Comment:
IIRC it only fails on the quarkus, so maybe we can only turn it off for quarkus-integration?
Either way, good for another PR, not an issues for this one.

Reply:
No, it fails for the optaplanner-core, too.